extensions/amp-consent/0.1/test/test-amp-consent.js

```diff
@@ -385,9 +385,59 @@ describes.realWin(
             'https://server-test-3/':
               '{"consentRequired": true, "consentStateValue": "unknown"}',
             'https://geo-override-check2/': '{"consentRequired": true}',
+            'https://gdpr-applies/':
+              '{"consentRequired": true, "gdprApplies": false}',
           };
         });
 
+        describe('gdprApplies value', () => {
+          it('uses given value', async () => {
+            const inlineConfig = {
+              'consentInstanceId': 'abc',
+              'consentRequired': 'remote',
+              'checkConsentHref': 'https://gdpr-applies/',
+            };
+            ampConsent = getAmpConsent(doc, inlineConfig);
+            await ampConsent.buildCallback();
+            await macroTask();
+            const stateManagerGdprApplies = await ampConsent
+              .getConsentStateManagerForTesting()
+              .getConsentInstanceGdprApplies();
+            expect(stateManagerGdprApplies).to.be.false;
+          });
+
+          it('defaults to consentRequired remote value', async () => {
+            const inlineConfig = {
+              'consentInstanceId': 'abc',
+              'consentRequired': 'remote',
+              'checkConsentHref': 'https://geo-override-check2/',
+            };
+            ampConsent = getAmpConsent(doc, inlineConfig);
+            await ampConsent.buildCallback();
+            await macroTask();
+            await expect(
+              ampConsent
+                .getConsentStateManagerForTesting()
+                .getConsentInstanceGdprApplies()
+            ).to.eventually.be.true;
+          });
+
+          it('defaults correctly when checkConsentHref is not defined', async () => {
+            const inlineConfig = {
+              'consentInstanceId': 'abc',
+              'consentRequired': false,
```

Comment:
Does the inline `consentRequired` value affect the `gdprApplies` value? 
My understanding is it won't, if so do you think we should set the value `true` instead to make more obvious that `gdprApplies` value is not related to the inline `consentRequired` value. 

(Sign.... two fields with the same name is soooo confusing)

Reply:
We use the inline `consentRequired` value when `gdprApplies` or `checkConsentHref` is not supplied. And since `consentRequired` will always be a valid value after `consent-config.js` merges it, it's a good fallback.

I've change the test name and values to reflect this.

Reply:
Updated PR description to reflect or decision (never fallback to local `consentRequired`)